pmm-app/src/pmm-qan/panel/components/Filters/components/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -62,13 +62,13 @@ export const CheckboxGroup: FC<CheckboxGroupProps> = ({
       return (
         <div
           className={styles.label}
-          key={`${group}:${item.value || ''}`}
+          key={`${group};${item.value || ''}`}
           data-qa={`filter-checkbox-${item.value}`}
         >
           <span className={styles.filterName}>
             <CheckboxField
               // TODO: using '--' because final form think that it is a nested fields
```

Comment:
is this a TODO or a NOTE?

Reply:
@nicolalamacchia I'd say it's NOTE, planned to replace it with something more elegant :D